src/components/Elements/Pages/MobilityRow.tsx

```diff
@@ -73,52 +80,69 @@ const MobilityRow = ({
                                     fontWeight: '600',
                                     color: colors.text,
                                     marginBottom: 1,
-                                    textAlign: 'center',
                                 }}
                                 numberOfLines={2}
                                 textBreakStrategy="highQuality"
                             >
-                                {item.route}
+                                {kind === 'bus' ? item.route : item.name}
                             </Text>
                         </View>
                         <Text
                             style={{
                                 fontSize: 16,
                                 color: colors.text,
+                                textDecorationLine:
+                                    item.canceled === true
+                                        ? 'line-through'
+                                        : 'none',
                             }}
                             numberOfLines={2}
                         >
                             {item.destination}
                         </Text>
                     </View>
-                    <View
+                )}
+                <View
+                    style={{
+                        flexDirection: 'column',
+                        alignItems: 'flex-end',
+                        justifyContent: 'center',
+                        padding: 5,
+                    }}
+                >
+                    <Text
                         style={{
-                            flexDirection: 'column',
-                            alignItems: 'flex-end',
-                            justifyContent: 'center',
-                            padding: 5,
+                            fontSize: 14,
+                            fontWeight: '600',
+                            color: colors.labelColor,
+                            marginBottom: 1,
                         }}
+                        numberOfLines={2}
+                        textBreakStrategy="highQuality"
                     >
-                        <Text
-                            style={{
-                                fontSize: 14,
-                                fontWeight: '600',
-                                color: colors.labelColor,
-                                marginBottom: 1,
-                            }}
-                            numberOfLines={2}
-                            textBreakStrategy="highQuality"
-                        >
-                            {timeString}
-                        </Text>
-                    </View>
+                        {kind === 'bus' || kind === 'train'
+                            ? timeString
+                            : infoString}
+                    </Text>
```

Comment:
Two different text components are used here for Bus/Train and Car/EV, which is why the font styles is different for the different views. Please fix this to achieve a consistent design.